### DISCUSSION TITLE
Dogecoin is not moving to PoS from PoW (afaik)

### DISCUSSION THREAD

**Discussion by GiverofMemory** [dogecoin/dogecoin#2783] [General]
Seen some unfortunate articles like this 

https://cryptopotato.com/dogecoin-foundation-joins-forces-with-vitalik-buterin-to-build-community-staking/

Where reporters got the wrong idea about community staking saying that dogecoin is leaving proof of work for proof of stake.  It is an honest mistake but one that may scare to death dogecoin miners, thinking that thier mining investment would be obsoleted.  Pretty much everyone in the crypto industry are reading these crypto articles and believing them.

As far as I know the PoS community staking mechanism would be a sidechain and not effect the Proof of Work mining.

I really wish reporters would do their due diligence but I guess from our perspective we should just be as detailed as possible in press releases so people don't get wrong ideas.  I do love the foundation making press releases, keep up the good work.

**Comment by tealethetoolman** [comment]
I wanted to add an argument of why PoW > Pos
With proof of work, in order to cheat the system, you would need to own a fab creating revolutionary hardware and a power source to run that hardware. Very high bar. Proof of stake, it just takes a rich person with a bunch of money to stake and point you at malicious nodes. Like if Janet Yellen told Jeff Bezos, “hey I need you to do a favor and end this once and for all”, it would be with PoS. 

Really, not sure why anyone is letting Vitalik Butter around with doge, he has his own network to destroy.

**Comment by qlpqlp** [comment]
A true minner and also coders, dont belives in the news that goes around, he belives on what is actually being done on codding. Its not like "Buy the Rumor", "Sell the news" lol

Also belive more on PoW then PoS maybe we can find a betther way then PoW and PoS :) we never know :)

**Reply by tealethetoolman** [reply]
Yeah i looked at his user and didn’t see any activity or forks. I hope they don’t come and try to be destructive and turn doge into xrp. I saw his name on the doge foundation website and that is was is raising all kinds of red flags for me. Doge is great because fees are low. No need to pay $100 per transaction.

**Comment by SoraKohaku** [comment]
This why make me confused as we know doge never PoS. Now my wallet doing PoS. So miner disadvantage I think. I'm mining too since algorithm not changed.
![Screenshot_20220312-200811](https://user-images.githubusercontent.com/11433397/158019211-97bb7153-a411-4dd1-a1f9-dc8b863b352b.png)

Did ever official announcement stake?

**Reply by patricklodder** [reply]
which wallet is this?

**Reply by SoraKohaku** [reply]
Triv.co.id

**Reply by patricklodder** [reply]
They call it "staking", but what they really do is lend out your DOGE to people that do margin trading. It's not staking, at all, it's lending.

**Reply by SoraKohaku** [reply]
I need carefully. So far I stake cardano and mining doge at pool. My asset check do at official
